ec2/spark_ec2.py

```diff
@@ -148,6 +160,10 @@ def parse_args():
         "-v", "--spark-version", default=DEFAULT_SPARK_VERSION,
         help="Version of Spark to use: 'X.Y.Z' or a specific git hash (default: %default)")
     parser.add_option(
+        "--tachyon-version",
+        default=DEFAULT_TACHYON_VERSION,
+        help="When --spark-version is a git hash, this will be used as the version of Tachyon")
```

Comment:
The wording here is a little confusing, I think. So this option only takes effect if `--spark-version` is a git hash and otherwise it is ignored, right?

And what's the purpose of the default value here? Doesn't seem to ever come into play.